recOrder/plugin/main_widget.py

```diff
@@ -190,6 +194,24 @@ def __init__(self, napari_viewer: Viewer):
         self.ui.qbutton_acq_phase_from_bf.clicked[bool].connect(
             self.acq_phase_from_bf
         )
+        self.bf_keywords = [
+            "bf",
+            "brightfield",
+            "bright",
+            "labelfree",
+            "label-free",
+            "lf",
+            "label",
+            "phase",
+            "ph",
+        ]
+        msg = (
```

Comment:
Why do we initialize this by joining the empty string with the message? AFAIK they are all run in the same `__init__()` method.

Reply:
I want to use the keywords elsewhere, and I want to append these to the message. Is there an easier way you'd suggest?

Reply:
Sorry I meant that you don't need this line:
https://github.com/mehta-lab/recOrder/blob/9bfce2833a3f17fc55f0891dbcc302ad1a89b983/recOrder/plugin/main_widget.py#L72
since it's equivalent if you just declare it here:
https://github.com/mehta-lab/recOrder/blob/9bfce2833a3f17fc55f0891dbcc302ad1a89b983/recOrder/plugin/main_widget.py#L213

Reply:
Aha! Fixed, thank you.